yt/geometry/grid_geometry_handler.py

```diff
@@ -418,7 +418,7 @@ def _chunk_io(
         if chunk_sizing == "auto":
             chunk_ngrids = len(gobjs)
             if chunk_ngrids > 0:
-                nproc = np.float(ytcfg.get("yt", "internals", "global_parallel_size"))
+                nproc = float(ytcfg.get("yt", "internals", "global_parallel_size"))
```

Comment:
This should probably be
```suggestion
                nproc = ytcfg.get("yt", "internals", "global_parallel_size")
```
or 
```suggestion
                nproc = int(ytcfg.get("yt", "internals", "global_parallel_size"))
```
so that it'd work even if `yt.internals.global_parallel_size == "1"`.

Reply:
The good news is that if you accept the change, the tests will be retriggered and hopefully pass the Eisenbug.